source/extensions/filters/network/dubbo_proxy/dubbo_protocol_impl.cc

```diff
@@ -21,10 +21,10 @@ constexpr uint64_t BodySizeOffset = 12;
 } // namespace
 
 // Consistent with the SerializationType
-bool isValidSerializationType(SerializationType type) {
+bool isValidDeserializationType(DeserializationType type) {
```

Comment:
I don't think you need this rename?

Reply:
I just want to keep the name of unity, it seems that reason is not sufficient.

Reply:
Serialization is the correct noun here, as well as those in config proto. I just wanted the verbs to be "deserialize" to match what they does.

Reply:
my bad,  fixed